quadriga/src/main/java/edu/asu/diging/quadriga/web/ListArchivedCollectionController.java

```diff
@@ -0,0 +1,48 @@
+package edu.asu.diging.quadriga.web;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.quadriga.core.data.CollectionRepository;
+
+@Controller
+public class ListArchivedCollectionController {
+
+    private Logger logger = LoggerFactory.getLogger(getClass());
+
+    @Autowired
+    private CollectionRepository collectionRepo;
```

Comment:
controllers should not directly talk to the repositories; there should be a service layer in between.